userguide/platform/userguide_configuration.adoc

```diff
@@ -62,178 +95,296 @@ curl -v \
+
+The following table lists the third party configuration files supported by Kill Bill and the corresponding system properties that need to be specified.
+
+[options="header",cols="1,1,1"]
+|===
+|Third-Party Configuration   |System Property   |Default Location
+//----------------------
+|ehcache   |org.killbill.cache.config.location   |https://github.com/killbill/killbill/blob/2b80b0445c7baf1f613425bb236a8cb36f1f377a/util/src/main/resources/ehcache.xml[ehcache.xml]   
+|logback   |logback.configurationFile|https://github.com/killbill/killbill/blob/2b80b0445c7baf1f613425bb236a8cb36f1f377a/profiles/killbill/src/main/resources/logback.xml[logback.xml]   
+|shiro |org.killbill.security.shiroResourcePath |https://github.com/killbill/killbill/blob/2b80b0445c7baf1f613425bb236a8cb36f1f377a/profiles/killbill/src/main/resources/shiro.ini[shiro.ini]   
+|===
+
+
+=== Environment Variables
+
+Kill Bill also allows defining configuration properties via environment variables. In such a case, the value specified via the environment variable overrides the value defined in the `killbill.properties` file.
+
+In order to specify a property via an environment variable, you need to define an environment variable corresponding to the underlying property and prefixed with *KB_*. For example, you can define an environment variable corresponding to the `org.killbill.dao.url` property as follows:
+[source,bash]
+----
+KB_org.killbill.dao.url = <jdbc URL here>
+----
+
+
+== Setup Specific Notes
+
+As explained in the https://docs.killbill.io/latest/getting_started.html[Getting Started] guide,  there are several ways to install Kill Bill such as https://docs.killbill.io/latest/getting_started.html#_aws_one_click[AWS], https://docs.killbill.io/latest/getting_started.html#_docker[Docker] or https://docs.killbill.io/latest/getting_started.html#_tomcat[Tomcat]. In addition, you may also be using Kill Bill for development/debugging using a https://docs.killbill.io/latest/development.html[standalone] setup.  
+
+This section provides some setup specific instructions such as location of the default `killbill.properties` and so forth.
+
+=== AWS
+
+In case of an AWS installation, the `killbill.properties` is present at the  `/var/lib/killbill/config/killbill.properties` location. 
+
+=== Docker
+
+In case Kill Bill is installed via Docker, the `killbill.properties` file is present at the `/var/lib/killbill/killbill.properties` path within the Docker container.  You can modify the properties defined here. 
+
+Alternatively, you can bind mount your own file, by specifying `-v /path/to/killbill.properties:/var/lib/killbill/killbill.properties` when starting your container.
+
+You can also specify configuration properties using individual environment variables.The following table lists all the environment variables and their default values:
+
```

Comment:
I just copied the content from the existing configuration document in this section. Does anything else need to be added here?

Reply:
So, do all the properties like `KILLBILL_DAO_URL` need to be replaced by their equivalent values (like `KB_org_killbill_dao_url`) ?

Reply:
I've updated the Docker images so that the legacy properties (e.g. `KILLBILL_DAO_URL`) still work. That way, our users don't have to change anything in their setup.

The new format (`KB_org_killbill_dao_url`) is preferred though.